cupy/math/misc.py

```diff
@@ -93,10 +95,10 @@ def _dot_convolve(a1, a2, mode):
     if a1.size == 0 or a2.size == 0:
         raise ValueError('Array arguments cannot be empty')
```

Comment:
Can I move this check to `correlate()` ? Already `convolve()` checks for empty inputs and also in order to keep consistency between `_fft_convolve` and `_dot_convolve`

Reply:
I agree your suggestion!